app/javascript/Search/Search.jsx

```diff
@@ -51,6 +51,25 @@ export class Search extends Component {
   componentDidMount() {
     this.registerGlobalKeysListener();
     InstantClick.on('change', this.enableSearchPageListener);
+
+    window.onpopstate = () => {
```

Comment:
I always forget, doesn't this override any other `onpopstate` that might there be set? Would it be better with `addEventListener`? Or are we overriding on purpose?

Reply:
I looked at the documentation. And that seems to be the way to do it. I did this late last night though. I'll double check, because adding an event listener is what I do usually.

Reply:
Yeah, so it has `addEventListener` for this. Just going to make the change.

Reply:
Cool :) It seems it can be done in both ways https://developer.mozilla.org/en-US/docs/Web/API/Window/popstate_event

Reply:
This is good to go now @rhymes. I added some tests to verify the event is added and removed.